docs/guides/protect-page-login/_common/teaser.mdx

```diff
@@ -1,21 +1,27 @@
 import VideoEmbed from '@site/src/components/VideoEmbed'
 
 <p>
-  In this guide, we set up a new {props.framework} application and protect its
-  home page with a login page using Ory. It will also add features such as
-  profile settings, registration, and more. You can use this guide with both Ory
-  Cloud and by self-hosting Ory software.
+This guide shows how to create a simple {props.framework} application and secure access to the app with Ory. You can use this guide with both Ory Cloud and self-hosted Ory software.
+
 </p>
 
-This guide is for you if you:
+
+This guide is perfect for you if:
 
 <ol>
-  <li>have {props.framework} installed;</li>
-  <li>want to use {props.framework} to build an application;</li>
-  <li>want to show your app only if the user is logged in.</li>
+
+<li>You have {props.framework} installed.</li>
+<li>You want to build an app using {props.framework}.</li>
+<li>You want to give access to your application to signed-in users only.</li>
+
 </ol>
```

Comment:
Please don't add newlines between HTML in MDX as they are interpreted as separate content blocks which are then wrapped in paragraphs, causing this to become invalid / broken HTML

Reply:
If I remember correctly, I had to add this new line - otherwise the thing wouldn't render properly.

Reply:
Okay, got rid of the newlines!